containers/attributes/main.py

```diff
@@ -108,7 +108,7 @@ def get_retryable_request():
 # This is a low priority though since it doesn't save us from having to get the
 # realmId first and so is a largely cosmetic difference
 async def get_idp_public_key(realm_id):
-    url = f"{os.getenv('OIDC_SERVER_URL')}/realms/{realm_id}"
+    url = f"{os.getenv('OIDC_SERVER_URL')}realms/{realm_id}"
```

Comment:
```suggestion
    url = f"{os.getenv('OIDC_SERVER_URL').rstrip('/')}realms/{realm_id}"
```

https://stackoverflow.com/questions/6584871/remove-last-character-if-its-a-backslash